shell/initialize/client.js

```diff
@@ -660,6 +694,9 @@ const noopData = () => {
 const noopFetch = () => {};
 
 // Special hot reload with asyncData(context)
+// still don't fully understand all of these functions, but doesn't vue offer hot reload?
+// do we need all of this complexity? Even now hot reload is not perfect and have to hit refresh sometimes
+// unless I am just clueless about the code, I don't undertand why we need this complexity
```

Comment:
Let's remove as much as we can but I'd like to avoid breaking hot reload any further.